src/definitions/index.ts

```diff
@@ -0,0 +1,59 @@
+import length from '../definitions/length';
+import area from '../definitions/area';
+import mass from '../definitions/mass';
+import volume from '../definitions/volume';
+import each from '../definitions/each';
+import temperature from '../definitions/temperature';
+import time from '../definitions/time';
+import digital from '../definitions/digital';
+import partsPer from '../definitions/partsPer';
+import speed from '../definitions/speed';
+import pace from '../definitions/pace';
+import pressure from '../definitions/pressure';
+import current from '../definitions/current';
+import voltage from '../definitions/voltage';
+import power from '../definitions/power';
+import reactivePower from '../definitions/reactivePower';
+import apparentPower from '../definitions/apparentPower';
+import energy from '../definitions/energy';
+import reactiveEnergy from '../definitions/reactiveEnergy';
+import volumeFlowRate from '../definitions/volumeFlowRate';
+import illuminance from '../definitions/illuminance';
+import frequency from '../definitions/frequency';
+import angle from '../definitions/angle';
+import charge from '../definitions/charge';
+import force from '../definitions/force';
+import acceleration from '../definitions/acceleration';
+import pieces from '../definitions/pieces';
+
+const allMeausures = {
```

Comment:
Purely from a maintenance point of view it might be nice to list these in alphabetical order. It then becomes easier to add/remove measures and also quickly scan through the list by eye and ensure all files are included. I would personally also sort the imports above alphabetically.